core.html

```diff
@@ -288,24 +288,26 @@ <h3>Method 3: DID Shortening with SHA-256 Hash</h3>
                 DID
                 identifier derived from a SHA-256 hash of the original identifier. The benefit of using Method 3 over Method to is the ability to have smaller size didcomm messages as did:peer2. dids tend to be verbose in nature. Method 3 peer dids can only be used after a peer did method 2 has been exchange with the other party and thus can map the shortened did to the longform one. In order to send a message encrypted with method 3 you first MUST send a discover-feature message (using the method 2 as the `to` field) to make sure that the receiving agent is capable of resolving method 3 dids.</p>
             <pre class="example" class="ABNF" title="ABNF for peer DIDs">
-peer-did-method-3 = "did:peer:3." 64*HEXDIG
+peer-did-method-3 = "did:peer:3" transform encnumbasis
+transform = "z"
+encnumbasis = 46*BASE58BTC
 </pre>
             <ul>
                 <li>Start with the DID generated using Method 2.</li>
                 <li>Take the SHA-256 hash of the generated DID (excluding the "did:peer:2" prefix).</li>
-                <li>Encode the hash using lowercase hexadecimal encoding.</li>
-                <li>Construct the final Method 3 DID by concatenating the prefix "did:peer:3" with the hex-encoded hash.
+                <li>Encode the hash using the base58btc multibase encoding.</li>
```

Comment:
To clarify, my suggestion in #50 was to use multibase + multicodec; multibase gives us the `z` on the front of the and the `multicodec` gives us two prefix bytes of `0x12 0x20` which indicate that the data is a sha2-256 hash 32 bytes in length. It's unclear whether that is adequately captured in these changes.